api/aws/iam/v1/iam.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package clutch.aws.iam.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/aws/iam/v1;iamv1";
+
+import "api/v1/annotations.proto";
+
+message Role {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.aws.iam.v1.Role",
+    pattern : "{account}/{region}/{name}"
+  };
+
+  string name = 1;
+  string id = 2;
+  string arn = 3;
+  string created_date = 4;
```

Comment:
should we use `google.protobuf.Timestamp` here?

Reply:
Make sense! Will update